cdap-conversion/src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,5 @@
+CDAP-Adapter-Type: streamConversion
+CDAP-Source-Type: STREAM
+CDAP-Sink-Type: DATASET
+CDAP-Scheduled-Program-Id: StreamConversionWorkflow
```

Comment:
We don't need the CDAP-Scheduled-Program-Id. All workflows in the application will be scheduled.